lib/galaxy/config/sample/job_conf.sample.yml

```diff
@@ -586,6 +586,23 @@ execution:
       # argument by default. You can turn this off by setting singularity_cleanenv to `false`.
       #singularity_cleanenv: true
 
+      # Singularity by default inherits the PID namespace, this can give
+      # issues with multiprocessing, hence galaxy passes the `--pid` argument
+      # by default to isolate the PID namespace. You can turn this off by setting
+      # singularity_pid to `false`.
+      #singularity_pid: true
+      Singularity by default inherits the IPC namespace, this can give
+      issues with multiprocessing, hence Galaxy passes the `--ipc` argument
+      by default to isolate the IPC namespace. You can turn this off by setting
+      singularity_ipc to `false`.
```

Comment:
```suggestion
      # Singularity by default inherits the IPC namespace, this can give
      # issues with multiprocessing, hence Galaxy passes the `--ipc` argument
      # by default to isolate the IPC namespace. You can turn this off by setting
      # singularity_ipc to `false`.
```